lib/usd/translators/lightWriter.cpp

```diff
@@ -222,4 +223,114 @@ void PxrUsdTranslators_AreaLightWriter::Write(const UsdTimeCode& usdTime)
         usdTime, primSchema, lightFn, _GetSparseValueWriter());
 }
 
+#ifdef UFE_VOLUME_LIGHTS_SUPPORT
+PXRUSDMAYA_REGISTER_WRITER(volumeLight, PxrUsdTranslators_VolumeLightWriter);
```

Comment:
One major problem I was struggling is here. As shown on L179, we should register a schema here but one Maya light type can only map to one usd type. Here there will be three possible schemas to register for volumeLight so I didn't register at the time. After playing around I dont see a major issue caused by this but I think we better address this.